src/main/java/Duke.java

```diff
@@ -1,10 +1,105 @@
+import java.util.Scanner;
+import Task.Deadline;
+import Task.ToDo;
+import Task.Events;
+import Task.Task;
+
 public class Duke {
+
+    static Task[] t = new Task[100];
+    static Scanner in = new Scanner(System.in);
+
+    public static void displayTaskList() {
+        System.out.println("    ____________________________________________________________\n" +
+                "     Here are the tasks in your list:");
+        for (int i = 0; i < Task.getNoOfTask(); i++) {
+            if(t[i].getTaskType().equals("Deadline")) {
+                System.out.println("     " + (i+1) + "." + t[i]);
+            } else if(t[i].getTaskType().equals("Events")) {
+                System.out.println("     " + (i+1) + "." + t[i]);
+            } else {
+                System.out.println("     " + (i+1) + "." + t[i]);
+            }
+        }
+        System.out.println("    ____________________________________________________________\n");
+    }
+
+    public static void taskMarkAsDone(int itemIndex) {
+        String taskDoneDescription = "";
+        for(int i = 0; i < Task.getNoOfTask(); i++) {
+            if(itemIndex == i) {
+                taskDoneDescription = t[i].getTaskDescription();
+                t[i].taskDone();
+            }
+        }
+        System.out.println("    ____________________________________________________________\n" +
```

Comment:
Consider creating a function for the line "____________________"  so that it will improve code clarity and readability